### PR TITLE
DST-1321 Introduces Storybook to the project

### PR DESCRIPTION
**Pull request by danieldelcore**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
     "postinstall": "preconstruct dev",
-    "dev:pretty-proptypes": "bolt w pretty-proptypes run dev",
     "lint": "yarn eslint \"./**/*.js\"",
     "validate": "yarn lint && yarn test && yarn flow",
     "build": "preconstruct build",
     "clean": "yarn delete:modules && yarn delete:dist",
     "delete:dist": "bolt ws exec --parallel -- rm -rf dist",
-    "delete:modules": "bolt ws exec --parallel -- rm -rf node_modules && rm -rf node_modules"
-  },
-  "bolt": {
```
Comment (danieldelcore) [inline package.json:20]: dropped bolt in favour of yarn workspaces.

File: package.json [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
+    "start": "run-p start:*",
+    "start:watch": "preconstruct watch",
+    "start:stories": "start-storybook -p 6006",
     "test": "jest",
     "test:watch": "jest --watch",
     "changeset": "changeset",
     "apply-changesets": "changeset version",
     "release": "yarn build && changeset publish",
     "postinstall": "preconstruct dev",
-    "dev:pretty-proptypes": "bolt w pretty-proptypes run dev",
```
Comment (danieldelcore) [inline package.json:12]: There is no dev script on the other side of this so removing

File: package.json [hunk 3]
```diff
@@ -1,27 +1,30 @@
 {
   "name": "extract-react-types-mono-repo",
   "version": "0.0.0",
+  "private": true,
```
Comment (danieldelcore) [inline package.json:4]: Added for preconstruct. Root level package jsons should be private

**Review by DarkPurple141 — APPROVED** [review]
LGTM

**Review by lol-russo — COMMENTED** [review]
Nice 👏

**Review by alexreardon — COMMENTED** [review]
LGTM

**Pull request merged**